Formula/zig.rb

```diff
@@ -15,6 +15,9 @@ class Zig < Formula
   depends_on "llvm"
 
   def install
+    ENV["CC"] = Formula["llvm"].opt_bin/"clang"
+    ENV["CXX"] = Formula["llvm"].opt_bin/"clang++"
+    ENV["LD"] = Formula["llvm"].opt_bin/"ld.lld"
```

Comment:
@SMillerDev Don't we usually set compilers with `fails_with [:clang, :gcc]` to force compiling with llvm's clang?

Also, it seems weird that there was an llvm dependency but it used to be able to be built with Apple native clang.

Reply:
Ignore this comment.